docs/eigenda/integrations-guides/dispersal/dispersal-api/ecdsa-keypair-authentication-flow.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 2
+---
+
+# ECDSA Keypair Authentication Flow
```

Comment:
This page can probably be pulled into the API overview page. It might be helpful to give this level of documentation of the `DisperseBlob` endpoint first, and then build on it with the `DisperseBlobAuthenticated` overview.

Reply:
Sounds good